src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1213,9 +1213,12 @@ static bool addTriggeredSubscriptions_withCache
     {
```

Comment:
Not really, but the branch worked for those tests. If it still works after all the recent changes, then we may add the connection to 1406.